test/searchModal/test.js

```diff
@@ -15,7 +15,10 @@
  *
  * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
  */
-define(['jquery', 'ui/searchModal', 'core/store', './mocks/searchStore'], function ($, searchModalFactory, store) {
+
+define(['jquery', 'ui/searchModal', 'core/store', './mocks/mocks'], function ($, searchModalFactory, store) {
+    const classTree = mockedClassTree;
```

Comment:
where is this variable defined?!
please not use global variables, never, even in unit tests